lib/logstash/inputs/stomp.rb

```diff
@@ -37,6 +37,12 @@ class LogStash::Inputs::Stomp < LogStash::Inputs::Base
   #Auto reconnect interval in seconds
   config :reconnect_interval, :validate => :number, :default => 30
 
+  # Enable TLS/SSL connection?
+  config :ssl, :validate => :boolean, :default => false
```

Comment:
This is a hard thing for users to do correctly and it also puts the configuration in too many places. Because of this, we try to offer a setting for plugins using SSL that users can specify a file (openssl or keystore format, if both are possible):

* [elasticsearch output `cacert`](https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch-cacert)
* [beats input `ssl_certificate_authorities`](https://www.elastic.co/guide/en/logstash/current/plugins-inputs-beats.html#plugins-inputs-beats-ssl_certificate_authorities)
* [http output `cacert`](https://www.elastic.co/guide/en/logstash/current/plugins-outputs-http.html#plugins-outputs-http-cacert)

We aren't consistent in naming, unfortunately, but we are trying to provide this setting explicitly for users.